src/commands/convert/generic2hdf.ts

```diff
@@ -0,0 +1,96 @@
+import { Command, Flags } from '@oclif/core'
```

Comment:
We should be able to use `src/commands/convert/index.ts` to make this the command for `saf convert`